sql/core/src/test/scala/org/apache/spark/sql/execution/ExchangeSuite.scala

```diff
@@ -132,4 +136,29 @@ class ExchangeSuite extends SparkPlanTest with SharedSQLContext {
     val projection2 = cached.select("_1", "_3").queryExecution.executedPlan
     assert(!projection1.sameResult(projection2))
   }
+
+  test("SPARK-26601: Make broadcast-exchange thread pool configurable") {
+    val sparkConf = new SparkConf()
+      .set(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER.key, "1")
+      .set("spark.driver.allowMultipleContexts", "true")
+    val tss = new TestSparkSession(sparkConf)
+    SparkSession.setActiveSession(tss)
+    assert(SQLConf.get.getConf(StaticSQLConf.MAX_BROADCAST_EXCHANGE_THREADNUMBER) === 1)
+
+    Future {
+      Thread.sleep(5*1000)
+    }(BroadcastExchangeExec.executionContext)
```

Comment:
For style, please take a look for https://github.com/databricks/scala-style-guide